docs/transforms/bin.md

```diff
@@ -253,6 +253,7 @@ The following named reducers are supported:
 * *first* - the first value, in input order
 * *last* - the last value, in input order
 * *count* - the number of elements (frequency)
+* *density* – the number of elements normalized to have total area of 1
```

Comment:
```suggestion
* *density* – the count or sum normalized by series (*z*)
```

(the "or sum" part supposes that we do the weighted version)

And at the bottom of the list, we can add:

> When using the *density* reducer with the binX or the binY transforms, the normalization factor is 1 divided by the bin step — this allows to draw histograms with a constant total area. Otherwise it is 1.